lib/features/home/data/sources/home_remote_data_source_impl.dart

```diff
@@ -0,0 +1,58 @@
+import '../models/event_model.dart';
+import '../models/home_model.dart';
+import 'home_remote_data_source.dart';
+
+class HomeRemoteDataSourceImpl implements HomeRemoteDataSource {
+  @override
+  Future<HomeModel> getEventsHome() async {
+    return Future.delayed(
+      const Duration(seconds: 3),
+      () => _createEventsHomeModel(),
+    );
+  }
+
+  HomeModel _createEventsHomeModel() {
+    return HomeModel(
+      profileImage:
+          "https://cnn-arabic-images.cnn.io/cloudinary/image/upload/w_400,c_scale,q_auto/cnnarabic/2019/06/28/images/130170.webp",
+      myEvents: [
+        _createMyEventModel(),
+        _createMyEventModel(),
+      ],
+      suggestedEvents: [
+        _createSuggestedEventModel(),
+        _createSuggestedEventModel(),
+      ],
+    );
+  }
+
+  EventModel _createMyEventModel() {
+    return EventModel(
```

Comment:
you should able to pass json and convert it to data model

Reply:
Done